### PR TITLE
Add configurable labels to autobumper

### PR DESCRIPTION
**Pull request by mpherman2**
The autobumper config now has a new field "labels" which you can use to add labels to the autobump PR

### CODE REVIEW

File: experiment/autobumper/bumper/bumper.go [hunk 1]
```diff
@@ -376,8 +378,11 @@ func (w HideSecretsWriter) Write(content []byte) (int, error) {
 // "images" contains the tag replacements that have been made which is returned from "updateReferences([]string{"."}, extraFiles)"
 // "images" and "extraLineInPRBody" are used to generate commit summary and body of the PR
 func UpdatePR(gc github.Client, org, repo string, images map[string]string, extraLineInPRBody, login, baseBranch, headBranch string, allowMods bool, prefixes []Prefix, versions map[string][]string) error {
```
Comment (chaodaiG) [inline experiment/autobumper/bumper/bumper.go:380]: Is this still used?

File: experiment/autobumper/bumper/bumper.go [hunk 2]
```diff
@@ -116,6 +116,8 @@ type Options struct {
 	RemoteName string `yaml:"remoteName"`
 	// The name of the branch that will be used when creating the pull request. If unset, defaults to "autobump".
 	HeadBranchName string `yaml:"headBranchName"`
+	// Optional list of labels to add to the bump PR
+	Labels []string `yaml:"labels"`
```
Comment (chaodaiG) [inline experiment/autobumper/bumper/bumper.go:120]: We probably have discussed about this before 😅 , is `omitempty` needed?

**Review by chaodaiG — COMMENTED** [review]
Awesome!

File: experiment/autobumper/bumper/bumper.go (hunk 2 above)
Comment (mpherman2) [reply experiment/autobumper/bumper/bumper.go:120]: I don't believe it is needed. The default value is []. I tested running without labels in the config and it worked as intended. I think omitempty helps when going from a golang struct -> json, but it doesn't seem to do anything when going from json -> struct.

File: experiment/autobumper/bumper/bumper.go (hunk 1 above)
Comment (mpherman2) [reply experiment/autobumper/bumper/bumper.go:380]: Not in the autobumper, and it is redundant; however it is exported, and I wanted to keep the change backwards compatible.
/assign @alvaroaleman 
Can we get rid of this?

Comment (chaodaiG) [reply experiment/autobumper/bumper/bumper.go:380]: Good thoughts!

**Review by chaodaiG — APPROVED** [review]
Approve since this is already backward compatible. 

/hold
If you want to clean it up in current PR let's wait for @alvaroaleman to confirm. Otherwise please document the purpose why keeping this function, would be even better log a warning for anyone who invoke this function with a deprecation date

File: experiment/autobumper/bumper/bumper.go (hunk 1 above)
Comment (alvaroaleman) [reply experiment/autobumper/bumper/bumper.go:380]: Yeah, we are still using this

**Pull request merged**